src/js/components/graph/indicator-overlay.jsx

```diff
@@ -0,0 +1,93 @@
+// This file defines the indicator overlay component - the popup that appears on
+//  on the device screen the first time a user's graph exceeds the max visible
+//  nodes, and illustrates the gesture to use to scroll the graph.
+import React from 'react'
+import Radium from 'radium'
+
+let IndicatorOverlay = React.createClass({
+
+  getInitialState() {
+    return {
+      visible: false // overlay is initially hidden
+    }
+  },
+
+  getStyles() {
+    return {
+      // CSS for the overlay
+      overlayContainer: {
+        position: 'absolute',
+        zIndex: 1500,
+        width: '100vw',
+        height: '100vh',
+        backgroundColor: '#000',
+        opacity: '0.7',
+        textAlign: 'center',
+        display: 'block',
+        transition: 'display 1s',
+        marginTop: '-112px'
+      },
+      // CSS for hidden overlay
+      overlayContainerHidden: {
+        display: 'none'
+      },
+      // white text/arrow illustration within overlay
+      indicatorImg: {
+        position: 'relative',
+        top: '80vh'
+      }
+    }
+  },
+
+  // hides overlay
+  hide() {
+    this.setState({visible: false})
+  },
+
+  // displays overlay, only when not already previously shown
+  show() {
+    if (!localStorage.getItem('indicator-overlay-drawn')) {
+      this.setState({visible: true})
+      localStorage.setItem('indicator-overlay-drawn', true)
+    }
+  },
+
+  // toggles show/hide overlay
+  toggle() {
+    if (this.state.visible) {
+      this.hide()
+    } else {
+      this.show()
+    }
+  },
+
+  render() {
+    let styles = this.getStyles()
+
+    let style = styles.overlayContainer
+
+    if (!this.state.visible) {
+      // When hiding overlay, change 'style' to overlayContainerHidden so that
+      // 'display' is overriden
+      style = Object.assign({}, style, styles.overlayContainerHidden)
+    }
+
+    return (
+      <div style={style} onTouchTap={this._handleClick}>
+        <img style={styles.indicatorImg} src="
+          /img/scroll_indicator_overlay2.png" />
```

Comment:
Try breaking the line before `src="` rather than after. The spaces (indentation) after the opening quotes break the link for me: 
![image](https://cloud.githubusercontent.com/assets/5739676/17521497/e30beba4-5e53-11e6-866e-063cbe64e57d.png)